cmd/emp/main.go

```diff
@@ -54,8 +55,19 @@ func (c *Command) PrintLongUsage() {
 }
 
 func (c *Command) FullUsage() string {
+	parts := make([]string, 0, 2)
```

Comment:
i was iffy on whether or not this is the best practice for this time of operation